stl/src/excptptr.cpp

```diff
@@ -79,7 +79,7 @@ namespace {
     _Ty& _Immortalize() { // return a reference to an object that will live forever
         static once_flag _Flag;
         alignas(_Ty) static unsigned char _Storage[sizeof(_Ty)];
-        if (_Execute_once(_Flag, _Immortalize_impl<_Ty>, &_Storage) == 0) {
+        if (!_Execute_once(_Flag, _Immortalize_impl<_Ty>, &_Storage)) {

```

Comment:
How about `using _BOOL = int` ?

Reply:
I'd rather not use `_BOOL`, which is a macro-ish name, or `_Bool`, because it's a keyword in C. I suppose other _truly_ ugly variations are available, like `_BoOl`  or `_BooL`?

Reply:
I'll go with comments then